app/insituIndicateurs/page.tsx

```diff
@@ -107,83 +164,170 @@ const InseeCode = () => {
+    try {
+      await navigator.clipboard.writeText(
+        DESCRIPTION_COLONNE_INDICATEUR + identifiantIndicateur,
+      );
+      setIsCopied(true);
+      setTimeout(() => setIsCopied(false), 2000);
+    } catch (err) {
+      console.error("Failed to copy text: ", err);
+    }
+  };
+
   return currentStep === "loading" ? (
-    <Title title={TITLE} />
-  ) : currentStep === "config" ? (
     <div>
       <Title title={TITLE} />
-      <Configuration>
-        <Instructions />
-      </Configuration>
+      <div className="loading">
+        <span className="loader"></span>
+      </div>
+      <Instructions />
       <MyFooter />
     </div>
-  ) : currentStep === "menu" ? (
-    <div>
-      <Title title={TITLE} />
-      {globalError && (
-        <div className="alert-error">
-          <div>
-            <span>Erreur</span> : {globalError}
-          </div>
-        </div>
-      )}
-      <div className="menu">
-        <div className="centered-column">
-          <h2>Compléter les champs vides</h2>
-          <p>
-            Lancer une recherche globale sur l&apos;ensemble des lignes
-            n&apos;ayant pas d'indicateur renseigné.
-          </p>
-          <button
-            className="primary"
-            onClick={() => {
-              updateIndicateurs(true);
-            }}
+  ) : (
+    currentStep === "menu" && (
+      <div>
+        <Title title={TITLE} />
+        <p>
+          Colonne sélectionnée :{" "}
+          <span className="tag validated semi-bold">
+            {mappings![COLUMN_MAPPING_NAMES.VALEUR_INDICATEUR.name]}
+          </span>
+        </p>
+        <p>
+          Consulter{" "}
+          <a
+            href="https://catalogue-indicateurs.donnees.incubateur.anct.gouv.fr/"
+            target="_blank"
+            title="Lien catalogue d'indicateur de l'ANCT - ouvre une nouvelle fenêtre"
           >
-            Première recherche
+            le catalogue d'indicateur de l'ANCT
+          </a>{" "}
+          pour trouver l'indicateur qui vous intéresse, copiez son identifiant
+          et collez le ci-dessous.
+        </p>
+        <label htmlFor="indicatorId">Identifiant de l'indicateur :</label>
+        <input
+          type="text"
+          id="indicatorId"
+          name="indicatorId"
+          placeholder="Entrez l'identifiant de l'indicateur"
+          defaultValue={identifiantIndicateur}
+          onChange={(event) => setIdentifiantIndicateur(event.target.value)}
+        />
+        <div className="advice">
+          <span className="text-to-copy">
+            Nous vous conseillons d'ajouter une description à votre colonne pour
+            ne pas perdre la trace de l'identifiant utilisé :
+            <br />
+            Exemple : « {DESCRIPTION_COLONNE_INDICATEUR}
+            <i>
+              {identifiantIndicateur.length > 0
+                ? identifiantIndicateur
+                : "renseigner un identifiant d'indicateur"}
```

Comment:
J'ai l'impression que le temps est plutôt l'impératif que l'infinitif ici, non ?

```suggestion
                : "renseignez un identifiant d'indicateur"}
```